src/PowerShell/Microsoft.WinGet.Client/Module/Microsoft.WinGet.Client.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package >
+  <metadata>
+    <id>Microsoft.WinGet.Client</id>
+    <version>0.1.0</version>
+    <authors>Microsoft</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/microsoft/winget-cli/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/microsoft/winget-cli</projectUrl>
+    <description>The Windows Package Manager PowerShell Module</description>
+    <copyright>$copyright$</copyright>

```

Comment:
This copyright variable is to be passed in? I do not see this will change a lot.